src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -37,4 +37,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageReference Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and !$(RuntimeFrameworkVersion.StartsWith('1.'))">
+    <PackageReference Include="Microsoft.NETCore.DotNetAppHost" Version="$(RuntimeFrameworkVersion)" IsImplicitlyDefined="true" />
```

Comment:
What is the point of having the `Microsoft.NETCore.DotNetHost` package anymore?  We will never use the `dotnet` executable from it, since standalone apps always use `DotNetAppHost`.

So instead of doing it this way, why don't we just put the `apphost` executable into the `DotNetHost` package?

Alternatively, change `Microsoft.NETCore.App` to reference `Microsoft.NETCore.DotNetAppHost` and remove the reference to `Microsoft.NETCore.DotNetHost`.